server/src/config.rs

```diff
@@ -31,6 +44,10 @@ impl Config {
         log::warn!("Creating Config object with default values.");
         Config {
             docker_port_grpc: 50051,
+            gis_host_grpc: String::from("svc-gis"),
```

Comment:
These new fields should be added to the tests below as well

Reply:
https://github.com/Arrow-air/svc-compliance/pull/11/commits/87b60b28c50e34eac6a06a06f1ae6476b4e291c1